ibm_mq/setup.py

```diff
@@ -0,0 +1,62 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from codecs import open  # To use a consistent encoding
+from os import path
+
+from setuptools import setup
+
+HERE = path.dirname(path.abspath(__file__))
+
+# Get version info
+ABOUT = {}
+with open(path.join(HERE, 'datadog_checks', 'ibm_mq', '__about__.py')) as f:
+    exec(f.read(), ABOUT)
+
+# Get the long description from the README file
+with open(path.join(HERE, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+
+CHECKS_BASE_REQ = 'datadog-checks-base'
+
+
+setup(
+    name='datadog-ibm_mq',
+    version=ABOUT['__version__'],
+    description='The Ibm_mq check',
+    long_description=long_description,
+    long_description_content_type='text/markdown',
+    keywords='datadog agent ibm_mq check',
+
+    # The project's main homepage.
+    url='https://github.com/DataDog/integrations-core',
+
+    # Author details
+    author='Datadog',
+    author_email='packages@datadoghq.com',
+
+    # License
+    license='BSD-3-Clause',
+
+    # See https://pypi.org/classifiers
+    classifiers=[
+        'Development Status :: 5 - Production/Stable',
+        'Intended Audience :: Developers',
+        'Intended Audience :: System Administrators',
+        'Topic :: System :: Monitoring',
+        'License :: OSI Approved :: BSD License',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+    ],
+
+    # The package we're going to ship
+    packages=['datadog_checks', 'datadog_checks.ibm_mq'],
```

Comment:
This should just be `packages=['datadog_checks', 'datadog_checks.ibm_mq'],` now https://github.com/DataDog/integrations-core/pull/2311

Reply:
I used the generator for this we should update it